src/os/bluestore/BitAllocator.h

```diff
@@ -364,6 +366,7 @@ class BitMapZone: public BitMapArea{
   }
 
   int64_t alloc_blocks(int64_t num_blocks, int64_t *start_block);
+  using BitMapArea::alloc_blocks_dis;
```

Comment:
Wait flag is not used currently as it is for a mode where we want to search for blocks without jumping a zone if another thread is already allocating in that zone.  So there are three modes a. ) where we keep searching and jump zones that are busy, b.) wait for the busy zone on the way to search, but still multiple threads can be active. c.) everything goes serial. I am sure this will be useful when we implement hint or at least we should not remove it till we are more clear on its usage.